### ISSUE TITLE
OSE Scaler Fails if first ansible master host is deleted & doesn't removes Master host from the Initial Nodes Section

### ISSUE THREAD

**Issue by philipakash** [aws-quickstart/quickstart-redhat-openshift#107]
There is a known issue in Open Shift which i think is still an open issue with 3.7 out.
Ref:
https://access.redhat.com/documentation/en-us/reference_architectures/2017/html/deploying_openshift_container_platform_3.5_on_amazon_web_services/failure_when_adding_node_s
We should handle this in `user data` or `ose_scaler` .

How to reproduce this ?
Just delete the masters[0] from the Auto Scaling Group and the Scaler fails as ca.serial.txt is not present in masters[1] and masters[2]
Red Hat recommends to copy the serial file to other masters before running the scaler playbook.

Another issue which i encountered is when the master is deleted by ASG OSE scaler won't remove old master host(one which got deleted) from the nodes section which fails the scaler playbook due to the unreachable host error.

**Comment by jaymccon** [comment]
@agamzn can you validate this please, if needed I'll be happy to get the ca files replicated to all the masters during bootstrapping.

**Comment by agamzn** [comment]
Circling back on this now that more-pressing commits are in place. Details pending.

**Comment by agamzn** [comment]
@philipakash , @jaymccon Apologies for the delay. I'm unable to reproduce this. :(

**Comment by philipakash** [comment]
@agamzn Can you check cat the serial files on openshift masters after relaunching the cluster ..you would find serial files in only one master.. Just delete the master which has the serial file and you should be able to reproduce it ..

**Comment by philipakash** [comment]
@agamzn I am testing the other fixes on my side ... Any updates on the master certificate bug ^

**Comment by agamzn** [comment]
That's dependent on #139. Apologies for the latency.

**Comment by pierreozoux** [comment]
I tried today with 3.11, and it still does fail :/
